dreamhome.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+from database.db import initialize_db
+
+app = Flask(__name__)
+
+app.config['MONGODB_SETTINGS'] = {
```

Comment:
Looks great! We will probably move this to a `.env` file so that we can differentiate our environments. 
- All the `.env` files would hold any "secret" information
- `.env` is most likely production
- `.env.test` is most likely test
- `.env.dev` is most likely dev

This way our secrets stay secret, but also, our databases stay separate as well